pkg/network/ebpf/c/protocols/mysql/helpers.h

```diff
@@ -0,0 +1,66 @@
+#ifndef __MYSQL_HELPERS_H
+#define __MYSQL_HELPERS_H
+
+#include "protocols/classification/common.h"
+#include "protocols/mysql/defs.h"
+#include "protocols/sql/helpers.h"
+
+// Validates the given buffer is of the format <number><delimiter> where the number is up to 2 digits.
```

Comment:
can you provide a link to the source of this statement? (the fact that each number in the version can contain up to 2 digits)